turbo-repo/apps/app/src/lang/es.json

```
@@ -1572,6 +1576,11 @@
       "detectColumns": "Detectar columnas",
       "addColumn": "Agregar columna",
       "cardLayout": "Diseño de Tarjeta",
+      "horizontal": "Horizontal",
+      "vertical": "Vertical",
+      "chartType": "Tipo de gráfico",
```

Biome (2.4.10)

[error] The key chartType was already declared.

(lint/suspicious/noDuplicateObjectKeys)